app/views/admin/settings/_mixed_settings_table.html.erb

```diff
@@ -37,7 +37,17 @@
         <% else %>
           <td colspan="2">
             <%= form_for(setting, url: admin_setting_path(setting), html: { id: "edit_#{dom_id(setting)}"}) do |f| %>
-              <%= f.text_area :value, label: false, id: dom_id(setting), lines: 1, class: "small-6 column" %>
+              <% if regional_setting?(setting) %>
+                <div class="small-6 column">
+                  <% if setting.key == "regional.time_zone.key" %>
+                    <%= f.select :value, time_zone_options_for_select(Setting["regional.time_zone.key"], ActiveSupport::TimeZone.all), label: false, id: dom_id(setting) %>
```

Comment:
Can we use here Time.zone.name instead of Setting["regional.time_zone.key"]? It seems that this is already loaded at `application.rb` boot at after_initialize block 🤔

Reply:
Same for line 45

Reply:
```suggestion
                    <%= f.select :value, time_zone_options_for_select(Time.zone.name, ActiveSupport::TimeZone.all), label: false, id: dom_id(setting) %>
```